Prusa-CW-Firmware/main.cpp

```diff
@@ -461,60 +492,58 @@ void setup() {
   menu_move(true);
 }
 
-
-void write_config(unsigned int address) {
-  EEPROM.put(address, magic);
-  address += MAGIC_SIZE;
-  EEPROM.put(address++, washing_speed);
-  EEPROM.put(address++, curing_speed);
-  EEPROM.put(address++, washing_run_time);
-  EEPROM.put(address++, curing_run_time);
-  EEPROM.put(address++, finish_beep_mode);
-  EEPROM.put(address++, drying_run_time);
-  EEPROM.put(address++, sound_response);
-  EEPROM.put(address++, curing_machine_mode);
-  EEPROM.put(address++, heat_to_target_temp);
-  EEPROM.put(address++, target_temp_celsius);
-  EEPROM.put(address++, target_temp_fahrenheit);
-  EEPROM.put(address++, SI_unit_system);
-  EEPROM.put(address++, heater_failure);
-  EEPROM.put(address++, FAN1_CURING_SPEED);
-  EEPROM.put(address++, FAN1_DRYING_SPEED);
-  EEPROM.put(address++, FAN1_PREHEAT_SPEED);
-  EEPROM.put(address++, FAN2_CURING_SPEED);
-  EEPROM.put(address++, FAN2_DRYING_SPEED);
-  EEPROM.put(address++, FAN2_PREHEAT_SPEED);
-  EEPROM.put(address++, resin_preheat_run_time);
-  EEPROM.put(address++, resin_target_temp_celsius);
+void write_config(unsigned int address) {		//useless address param
```

Comment:
main.cpp:495:32: warning: unused parameter 'address' [-Wunused-parameter]
 void write_config(unsigned int address) {  //useless address param

Please fix this compiler warning. (Remove unused function parameter)